MetaMask/metamask-extension
# UX: Provide entrypoint for survey

## **Description**

Adds the Survey toast to the home screen

## **Related issues**

Fixes: https://github.com/MetaMask/MetaMask-planning/issues/1650

## **Manual testing steps**

1. Open the home screen
2. See the survey
3. Click the close link
4. Refresh the extension
5. Don't see the survey again

## **Screenshots/Recordings**

### **Before**

N/A

### **After**
<img width="369" alt="SCR-20231121-lkiy" src="https://github.com/MetaMask/metamask-extension/assets/46655/9f4acf85-3cbf-4a61-86ce-8030e65402bb">


## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
